https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=278837#c3

(...)
2) It's our ports framework that is problematic. It comments out the "git" line in the workspace (dunno why). You can see it by yourself with:
make extract
cp xxx/Cargo.toml xxx/Cargo.toml.1
make configure
diff -dup xxx/Cargo.toml.1 xxx/Cargo.toml

--- Cargo.toml.orig	2024-08-07 22:07:24 UTC
+++ Cargo.toml
@@ -38,7 +38,7 @@ flate2            = { version = "1.0" }
 crossbeam-channel = { version = "0.5.12" }
 directories       = { version = "4.0.1" }
 flate2            = { version = "1.0" }
-git2              = { version = "0.19.0", features = ["vendored-openssl"] }
+git2              = { version = "0.19.0" }
 globset           = { version = "0.4.14" }
 hashbrown         = { version = "0.14.5", features = ["serde"] }
 im                = { version = "15.0.0", features = ["serde"] }
@@ -66,6 +66,11 @@ zstd              = { version = "0.11.2" }            
 toml_edit         = { version = "0.20.2", features = ["serde"] }
 url               = { version = "2.5.0" }
 zstd              = { version = "0.11.2" }                                                                         # follow same version wasmtime-cache in lockfile
+tracing           = { git = "https://github.com/tokio-rs/tracing", rev = "908cc432a5994f6e17c8f36e13c217dc40085704", package = "tracing" }
+tracing-log       = { git = "https://github.com/tokio-rs/tracing", rev = "908cc432a5994f6e17c8f36e13c217dc40085704", package = "tracing-log" }
+tracing-subscriber = { git = "https://github.com/tokio-rs/tracing", rev = "908cc432a5994f6e17c8f36e13c217dc40085704", package = "tracing-subscriber" }
+tracing-appender = { git = "https://github.com/tokio-rs/tracing", rev = "908cc432a5994f6e17c8f36e13c217dc40085704", package = "tracing-appender" }
+alacritty_terminal = { git = "https://github.com/alacritty/alacritty", rev = "cacdb5bb3b72bad2c729227537979d95af75978f", package = "alacritty_terminal" }
 
 lsp-types = { version = "0.95.1", features = ["proposed"] }                                                  # not following semver, so should be locked to patch version updates only
 psp-types = { git = "https://github.com/lapce/psp-types", rev = "f7fea28f59e7b2d6faa1034a21679ad49b3524ad" }
@@ -85,31 +90,6 @@ regalloc2 = { rev = "5d79e12d0a93b10fc181f4da409b4671d
 # Temporarily patch lsp-types with a version that adds message-type debug
 lsp-types = { git = "https://github.com/lapce/lsp-types", rev = "feaa1e2ec80975c9dadd400a238ceacf071058e6" }
 regalloc2 = { rev = "5d79e12d0a93b10fc181f4da409b4671dd365228", git = "https://github.com/bytecodealliance/regalloc2" }
-
-
-[workspace.dependencies.tracing]
-git     = "https://github.com/tokio-rs/tracing"
-rev     = "908cc432a5994f6e17c8f36e13c217dc40085704"
-package = "tracing"
-
-[workspace.dependencies.tracing-log]
-git     = "https://github.com/tokio-rs/tracing"
-rev     = "908cc432a5994f6e17c8f36e13c217dc40085704"
-package = "tracing-log"
-
-[workspace.dependencies.tracing-subscriber]
-git     = "https://github.com/tokio-rs/tracing"
-rev     = "908cc432a5994f6e17c8f36e13c217dc40085704"
-package = "tracing-subscriber"
-
-[workspace.dependencies.tracing-appender]
-git     = "https://github.com/tokio-rs/tracing"
-rev     = "908cc432a5994f6e17c8f36e13c217dc40085704"
-package = "tracing-appender"
-
-[workspace.dependencies.alacritty_terminal]
-git = "https://github.com/alacritty/alacritty"
-rev = "cacdb5bb3b72bad2c729227537979d95af75978f"
 
 [workspace.dependencies.windows-sys]
 version  = "0"
